microsoftgraph/microsoft-graph-explorer-v4
# Fix: Failing link display

## Overview
Queries with urls in the tip message fail when there are numeric values in the message. 

Fixes #899 
Fixes #892 

### Demo

N/A

### Notes

Made sure to only try and replace the links using the dollar sign

## Testing Instructions

* Run Graph Explorer
* Go to "Sample queries"
* Expand "Applications"
* Click on "retrieve application properties"
* See no error
